lib/frigg/persister.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Frigg
+  # Persister for Postgres MetadataAdapter.
+  class Persister < Valkyrie::Persistence::Fedora::Persister
+    # Persists a resource within the database
+    #
+    # Modified from the upstream to skip previously persisted check
+    #
+    # @param [Valkyrie::Resource] resource
+    # @return [Valkyrie::Resource] the persisted/updated resource
+    # @raise [Valkyrie::Persistence::StaleObjectError] raised if the resource
+    #   was modified in the database between been read into memory and persisted
+    # rubocop:disable Lint/UnusedMethodArgument
+    def save(resource:, external_resource: false, perform_af_validation: false)
+      # TODO
+      #   orm_object = resource_factory.from_resource(resource: resource)
+      #   orm_object.transaction do
+      #     orm_object.save!
+      #     if resource.id && resource.id.to_s != orm_object.id
+      #       raise Valkyrie::Persistence::UnsupportedDatatype,
+      #             "Postgres' primary key column can not save with the given ID #{resource.id}. " \
+      #             "To avoid this error, set the ID to be nil via `resource.id = nil` before you save it. \n" \
+      #             "Called from #{Gem.location_of_caller.join(':')}"
+      #     end
+      #   end
+      #   resource_factory.to_resource(object: orm_object)
+      # rescue ActiveRecord::StaleObjectError
+      #   raise Valkyrie::Persistence::StaleObjectError, "The object #{resource.id} has been updated by another process."
```

Comment:
This looks like Frigg isn't persisting?

Reply:
thats correct. it's structure is there, but there wasn't a fcrepo 6 version of Hyrax at the time I first put up the PR. I had a few requests to at least put the skeleton of an fcrepo 6 version in place even if it wasn't working yet, in order to signal to the community that it could be done. Once this is merged, we can do an investigation on getting fcrepo 6 version fully working. To be clear, I dont have any projects at the moment with fcrepo 6 on their road map.